sbysrc/sby_core.py

```diff
@@ -312,15 +320,15 @@ def parse_config(self, f):
                 if entries[0] == "files":
                     mode = "files"
                     if len(entries) != 1:
-                        self.error(f"sby file syntax error: {line}")
+                        self.error(f"sby file syntax error: '[files]' section expects no arguments, got '{line}'")
                     continue
 
-                self.error(f"sby file syntax error: {line}")
+                self.error(f"sby file syntax error: unexpected section '{entries[0]}', expected one of 'options, engines, script, autotune, file, files'")
 
             if mode == "options":
                 entries = line.split()
                 if len(entries) != 2:
-                    self.error(f"sby file syntax error: {line}")
+                    self.error(f"sby file syntax error: '[options]' section entry does not have an argument '{line}'")
```

Comment:
This message is a little awkward, but talking about it with @jix we probably want to change the behavior, not just the message:
- split only on the first space, not every space (thus enabling `expect pass, fail` to work)
- then only have a "too few arguments" syntax error.

Reply:
okay,sgtm